java/arcs/core/storage/BUILD

```diff
@@ -45,6 +45,7 @@ arcs_kt_library(
         "//java/arcs/core/storage/util",
         "//java/arcs/core/type",
         "//java/arcs/core/util",
+        "//third_party/java/androidx/annotation",
```

Comment:
You can't dep on Android libraries from core.

Reply:
oh, right, thanks.
is there an equivalent for `@VisibleForTesting` that i can use in core?

Reply:
The pattern we follow in core is to use `/* internal */` as a pseudo-modifier on the function signature, we can remove the `/* */` once bazel has better support for 'friends'.

Reply:
done, thanks